### PR TITLE
2023 Challenge Updates

### PR DESCRIPTION
**Pull request by Lucaweihs**
Fixes the openness issues in #40 by:

1. Now, issuing open commands with relevant object types visible will open/close those objects even if they were not originally different in the walkthrough/unshuffle phases.
2. New THOR version fixes issues with objects "opening" without any visual change.

Note that we do not address point 3 in #40 as it's both (1) rare and (2) the agent can, in principle, attempt to find an angle from which only a single object of the given type is visible.

This update also now includes new 2023 datasets and links to the new leaderboards.

### CODE REVIEW

**Comment by jordis-ai2** [comment]
LGTM

**Review by jordis-ai2 — APPROVED** [review]
Other than the data files, which I haven't checked, everything else looks clean to me 👍

File: rearrange/tasks.py [hunk 1]
```diff
@@ -97,7 +101,50 @@ def __init__(
...
+                    openness_1 = (
+                        seeded_rand.uniform(a, b)
+                        if seeded_rand.random() < (b - a) / ((b - a) + (d - c))
+                        else seeded_rand.uniform(c, d)
+                    )
+                else:
+                    priority_onames.append(up["name"])
+                    openness_1 = gp["openness"]
+
+                self.openable_obj_name_to_openness_iter[up["name"]] = (lambda a, b: iter(
```
Comment (jordis-ai2) [inline rearrange/tasks.py:133]: Not sure what this does tbh.

Comment (jordis-ai2) [inline rearrange/tasks.py:133]: Not sure what this does tbh.

Comment (jordis-ai2) [reply rearrange/tasks.py:133]: I believe it must be correct, but a comment might make it easier to follow.

Comment (Lucaweihs) [reply rearrange/tasks.py:135]: Commented! And that's exactly correct, just a needlessly complex way to toggle between values :)

**Pull request merged**